machineconfig/master/spec/config/storage/files/haproxy.cfg.template L2V0Yy9oYXByb3h5L2hhcHJveHkuY2ZnLnRlbXBsYXRlCg==

```diff
@@ -0,0 +1,22 @@
+global
```

Comment:
I think it would be good to add stats:

```
listen haproxy_stats :50000
stats enable
stats uri /
no log
```

I'm fine with doing it in a separate PR

Reply:
Another conf example for stats https://www.datadoghq.com/blog/how-to-collect-haproxy-metrics/